client/webserver/site/src/css/market.scss

```diff
@@ -63,24 +63,33 @@ table.ordertable {
   margin-bottom: 10px;
 
   th {
-    font-size: 14px;
     font-family: 'demi-sans', sans-serif;
   }
 
-  td {
-    font-size: 14px;
-  }
-
   td,
   th {
-    font-size: 14px;
+    font-size: 13px;
     padding: 3px 0;
   }
 
   tr {
     border-bottom: 1px solid #b3b3b3;
   }
 
+  thead,
+  tr {
+    display: table;
+    width: 100%;
+    table-layout: fixed;
+  }
+
+  tbody {
+    display: block;
+    width: 100%;
+    overflow: scroll;
+    table-layout: fixed;
```

Comment:
Can you also review the changes in the various responsive views, particularly the most narrow where all the tables and the order form are on top of each other?  In that view it seems like the tables are reserving more vertical space than before, but I could be mistaken.

Reply:
Before they just took up as much space as the rows in the table took, but I'll make them take up the same amount of space as the "Your Orders" table.

Reply:
"Before" was ok right?  Now when I'm in the narrow view they take a ton of space for nothing.  Not for you?

Reply:
Check it out now, see how you like it. It's the same smaller space in every case.